pkg/names/names.go

```diff
@@ -41,6 +41,12 @@ const CreateWaitAnnotation = "networkoperator.openshift.io/create-wait"
 // that they are not critical to the functioning of the pod network
 const NonCriticalAnnotation = "networkoperator.openshift.io/non-critical"
 
+// GenerateStatusLabel can be set by the various Controllers to tell the
+// StatusController that this object is relevant, and should be included
+// when genetating status from deployed pods.
+// Currently, this is looked for on Deployments, DaemonSets, and StatefulSets
+const GenerateStatusLabel = "networkoperator.openshift.io/generate-status"
```

Comment:
"generate-status" sounds like it means "someone should generate status for this pod". I'd suggest something more "deployed by CNO" / "tracked by CNO" ish.

Reply:
well, I was trying to make this explicit: this is consumed by the status controller. I could imagine naming this "networkoperator/operand-of=$controller-name", which would be cute, but doesn't specify what it **does**.

Reply:
> well, I was trying to make this explicit: this is consumed by the status controller.

Yeah, but I'm saying that "generate-status" doesn't imply that to me at all. Especially, it's not clear that the "status" there refers to _someone else's_ status, not the object's own status.

Reply:
How about (all namespaced with `networkoperator.openshift.io`
1. `controller=$controller`
2. `managed-by=$controller`
3. `operand-of=$controller`

Reply:
Leaning towards `networkoperator.openshift.io/generates-operator-status`.

Reply:
OK, updated based on this discussion.